.github/ISSUE_TEMPLATE.md

```diff
@@ -1,6 +1,19 @@
-## SP 800-63-3 Public Preview Has Closed!
+### All Fields Are Required
 
-Please check back in the coming weeks for the launch of our next public comment phase, where collection of issues and comments will resume. We may have even resolved the issue you were planning on submitting by then - and if not - please submit when we commence.  Until then, all issues received will be automatically closed.
+**Organization**:
 
-Thank you for your interest in supporting and improving NIST SP 800-63-3.
+**Organization Name (self, if individual):**
+ 
+**Type**:
+ 
+**Document (63-3, 63A, 63B, or 63C)**:
 
+**Reference (Include section and paragraph number)**:
+ 
+**Comment (Include rationale for comment)**:
+ 
+**Suggested Change**:
+ 
+---
+ 
+Organization: 1 = Federal, 2 = Industry, 3 = Other 
```

Comment:
Does this go with line 3? If so, it might be good to put it up there, e.g.:

**Organization** (1 = Federal, 2 = Industry, 3 = Other):

A lot of people seemed not to understand this on the preview template.